app/controllers/application_controller.rb

```diff
@@ -21,7 +17,7 @@ def current_user_uuid
                             "the #{Rails.env} environment.")
         end
 
-        OpenStax::Auth::Strategy2.user_uuid(request) || TEMP_ANONYMOUS
+        OpenStax::Auth::Strategy2.user_uuid(request)
```

Comment:
@reedstrm - side note - previously you had used `|| nil` at the end of these lines -- that is unnecessary because that `|| nil` clause would only have been reached if the first part of the line was already `nil`, and `nil || nil` is repetitive.